packages/dx-react-scheduler/src/plugins/day-view.tsx

```diff
@@ -179,23 +207,31 @@ class DayViewBase extends React.PureComponent<VerticalViewProps, ViewState> {
       appointmentLayerComponent: AppointmentLayer,
       cellDuration,
       name: viewName,
+      intervalCount,
+      startDayHour,
+      endDayHour,
     } = this.props;
-    const { timeTableRef: stateTimeTableRef } = this.state;
+    const { rects } = this.state;
 
     return (
       <Plugin
         name="DayView"
       >
-        <Getter name="availableViewNames" computed={this.availableViewNamesComputed} />
-        <Getter name="currentView" computed={this.currentViewComputed} />
-        <Getter name="intervalCount" computed={this.intervalCountComputed} />
-        <Getter name="cellDuration" computed={this.cellDurationComputed} />
-        <Getter name="viewCellsData" computed={this.viewCellsDataComputed} />
+        <Getter name="availableViewNames" computed={this.memoAvailableViewNames(viewName)} />
+        <Getter name="currentView" computed={this.memoCurrentView(viewName)} />
+
+        <Getter name="intervalCount" computed={this.memoIntervalCount(viewName, intervalCount)} />
+        <Getter name="cellDuration" computed={this.memoCellDuration(viewName, cellDuration)} />
+        <Getter
+          name="viewCellsData"
+          computed={this.memoViewCellsData(viewName, startDayHour, endDayHour, cellDuration)}
+        />
         <Getter name="startViewDate" computed={this.startViewDateComputed} />
         <Getter name="endViewDate" computed={this.endViewDateComputed} />
-        <Getter name="timeTableElement" computed={this.timeTableElement} />
-        <Getter name="layoutElement" computed={this.layoutElement} />
-        <Getter name="layoutHeaderElement" computed={this.layoutHeaderElement} />
+
+        <Getter name="timeTableElement" computed={this.memoTimeTableElement(viewName)} />
+        <Getter name="layoutElement" computed={this.memoLayoutElement(viewName)} />
+        <Getter name="layoutHeaderElement" computed={this.memoLayoutHeaderElement(viewName)} />
```

Comment:
Memoization is expected to be transparent - calling side should not distinguish if it calls memoized function or not.

Not
```jsx
<Getter name="something" computed={this.memoGetSomethingCalculator(...)} />
```

But
```jsx
<Getter name="something" computed={this.getSomethingCalculator(...)} />
```

----
Consequently - there is no need to keep original function together with memoized function in component instance.

Not
```jsx
getSomethingCalculator = (...) => { 
  ...
};
...
memoGetSomethingCalculator = memoize(this.getSomethingCalculator);
```

But
```jsx
getSomethingCalculator = memoize((...) => {
  ...
});
```